src/TLuaInterpreter.cpp

```diff
@@ -5182,6 +5182,23 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+// returns search paths for LuaGlobal itself to look at when loading other modules
+// follows the principle of closest paths to the binary first, furthest away lasy
+int TLuaInterpreter::getMudletLuaDefaultPaths( lua_State * L )
+{
+    lua_newtable( L );
+    lua_createtable(L,2,0);
+    // add filepath relative to the binary itself (one usecase is AppImage on Linux)
+    QString nativePath = QDir::toNativeSeparators( QCoreApplication::applicationDirPath() + "/mudlet-lua/lua/" );
```

Comment:
The macOS code only worked, because of the run_mudlet workaround, which was necessary, because, without that, the working directory is `/`. I only understood why, because you practically put the solution under my nose. We should be able to get rid of the workaround for good if we add the conditional compile in.

Otherwise, I'll make the changes and send in an extra PR if that's your preferred way.